src/screens/admin-home/components/header/styles.ts

```diff
@@ -0,0 +1,19 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
```

Comment:
After the new styles approach, I guess this has to be updated.

Reply:
Yes. `useStyles` + `useMemo`